packages/react/src/profile/useProfile.ts

```diff
@@ -37,16 +42,17 @@ export function useProfile({
   );
 
   const { data, error, loading } = useReadResult(
-    useGetProfile(
-      useLensApolloClient(
+    useGetProfile({
+      ...useLensApolloClient(
         useActiveProfileAsDefaultObserver({
           variables: useSourcesFromConfig({
             request,
             observerId,
           }),
         }),
       ),
-    ),
+      skip: request.skip,
```

Comment:
We should pass `skip` through `useActiveProfileAsDefaultObserver` or we risk to inhibit some of it's use cases.

```ts
      useLensApolloClient(
        useActiveProfileAsDefaultObserver({
          variables: useSourcesFromConfig({
            request,
            observerId,
          }),
          skip: request.skip,
        }),
      ),
```